cves/kernel/CVE-2013-5634.yml

```diff
@@ -151,10 +154,15 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Very little information is given on the discovery of this vulnerability. There
+    is no mention of who or how it was discovered in the patch commit. It appears
+    as though the vulnerability was introduced by the initial implementation of an
+    interface so it was likely discovered by a developer during testing or further
+    implementation.
+  automated: false
+  contest: false
+  developer: true
```

Comment:
Based on the instructions I was under that this flag was what the google developer instruction was referring to. Its possible I'm wrong but I thought I should mention it

Reply:
A google developer was involved in the discovery of this vulnerability but I failed to mention that in the answer, thanks for catching that!